docs/release-issue-template.md

```diff
@@ -38,7 +38,7 @@ You can see the proxy version on prod [on grafana](https://lightbendcloud.grafan
 
 ### Announcements
 
-- [ ] Announce in [the forum](https://discuss.lightbend.com/c/kalix/40)
-  - tag with Kalix, releases and announcement
+- [ ] Announce in [the forum](https://discuss.kalix.io/)
+  - tag with releases and announcement
```

Comment:
Don't think these tags already exist on the new forum but probably worth creating them apart from the `kalix` one?